activemodel/lib/active_model/errors.rb

```diff
@@ -398,21 +398,19 @@ def generate_message(attribute, type = :invalid, options = {})
       type = options.delete(:message) if options[:message].is_a?(Symbol)
 
       if @base.class.respond_to?(:i18n_scope)
-        defaults = @base.class.lookup_ancestors.map do |klass|
-          [ :"#{@base.class.i18n_scope}.errors.models.#{klass.model_name.i18n_key}.attributes.#{attribute}.#{type}",
-            :"#{@base.class.i18n_scope}.errors.models.#{klass.model_name.i18n_key}.#{type}" ]
+        i18n_scope = @base.class.i18n_scope.to_s
```

Comment:
I did it intentionally because `.to_s` is called in a loop. In order not to convert symbol to string over and over again, I decided to move it up. I'm not attached to it, I can remove it if my argument is not convincing enough.